.travis.yml

```diff
@@ -1,12 +1,37 @@
 language: c
 sudo: false
-compiler:
-- clang
-- gcc
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+    - os: linux
+      compiler: clang
+#    - os: osx
+#      compiler: gcc
+    - os: osx
+      compiler: clang
+# This section is required because the default Travis environment as of
+# April 2016 has CMake 2.8.7 installed.
+#addons:
+#  apt:
+#    sources:
+#    # contains CMake 2.8.12
+#    - kubuntu-backports
+#    packages:
+#    - cmake
 script:
-- ./autogen.sh
-- ./configure
-- make distcheck
+  # Print all environment variables to aid in CI development
+  - printenv
+  # Print version and available CMake generators to aid in CI development
+  - cmake --version
+  - cmake --help
+  # Perform out-of-source build
+  - mkdir build
+  - cd build
+  # Perform CMake backend generation, build, and test
+  - cmake -DCMAKE_C_COMPILER=$CC ..
```

Comment:
Is the definition of -DCMAKE_C_COMPILER required here?  (It doesn't pick up $CC automatically?)